fms_fsdp/utils/dataloader_utils.py

```diff
@@ -1,12 +1,13 @@
 import torch
 
 from fms_fsdp.utils.dataset_utils import (
-    Buffer_Dataset,
-    Checkpoint_Dataset,
-    Preload_Buffer_Dataset,
-    Preprocess_Dataset,
-    Sampling_Dataset,
-    Scalable_Shard_Dataset,
+    BufferDataset,
```

Comment:
might be better to do the reformat in a separate PR to make this particular logic change easier to find/read

Reply:
Oh yeah this is really only meant to be reviewed in context of #99. I can open a new PR without this dependency, but it'll introduce conflicts later on, and #99 is a large enough set of code changes as is without that complication